python/tests/stress/arcticdb/version_store/test_mem_leaks.py

```diff
@@ -177,9 +177,9 @@ def proc_to_examine():
          run the test from command line again to assure it runs ok before commit 
 
     """
-    max_mem_bytes = 175_623_040
```

Comment:
(Not for this PR) We should consider putting such global configurations in GitHub environment variables so that we can tweak them without having to to a PR.

Reply:
But it's important local runs match the CI runs?